docs/references/javascript/organization/invitations.mdx

```diff
@@ -40,16 +63,9 @@ Retrieve the members of the selected organization.
 
 ### Returns
 
-<Tables
-  headings={["Type", "Description"]}
-  headingsMeta={[{}, {maxWidth: "300px"}]}
-  rows={[
-  {cells: [
-  <code>Promise{"<"}<a href="/docs/references/javascript/organization-invitation">OrganizationInvitation</a>[]{">"}</code>,
-  <>This method returns a <code>Promise</code> which resolves with a list of <code><a href="/docs/references/javascript/organization-invitation">OrganizationInvitation</a></code> objects.</>,
-  ]},
-  ]}
-/>
+| Type | Description |
+| --- | --- |
+| <code>Promise{"<"}<a href="/docs/references/javascript/types/paginated-response">PaginatedResponse</a>{"<"}<a href="/docs/references/javascript/organization-invitation">OrganizationInvitation</a>{">"}</code> | This method returns a <code>Promise</code> which resolves with a list of <code><a href="/docs/references/javascript/organization-invitation">OrganizationInvitation</a></code> objects.|
```

Comment:
I would refactor to use reference links, like we do on /organization/domains 
 and /organization-domain

Don't forget to add the links at the bottom of the page ( [org-inv-ref]: /docs/references/javascript/organization-invitation)

```suggestion
| <code>Promise\<[PaginatedResponse][pag-res-ref]\<[OrganizationInvitation][org-inv-ref]\>></code> | This method returns a `Promise` which resolves with a list of [`OrganizationInvitation`][org-inv-ref] objects. |
```